include/z64cutscene_commands.h

```diff
@@ -11,445 +11,273 @@
- * FORMAT
- *   Capital U is Unused
- *   uuqqssss eeeeUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU 00000000 00000000 00000000
- *   size = 0x30
+ * Stops a sequence at the specified time.
+ * @note The sequence ID is subtracted by 1 before being used. Add +1 to the desired sequence ID when passing it in.
  */
-#define CS_STOP_BGM(sequence, startFrame, endFrame, unused0, unused1, unused2, unused3, unused4, unused5, unused6, unused7) \
-    CMD_HH(sequence, startFrame), CMD_HH(endFrame, unused0), \
-    CMD_W(unused1), CMD_W(unused2), CMD_W(unused3), CMD_W(unused4), CMD_W(unused5), \
-    CMD_W(unused6), CMD_W(unused7), 0x00000000, 0x00000000, 0x00000000
+#define CS_STOP_SEQ(seqId, frame, unused0, unused1, unused2, unused3, unused4, unused5, unused6, unused7, unused8) \
+    CMD_HH((seqId + 1), frame), CMD_HH(unused0, unused1), \
+    CMD_W(unused2), CMD_W(unused3), CMD_W(unused4), CMD_W(unused5), CMD_W(unused6), \
+    CMD_W(unused7), CMD_W(unused8), 0x00000000, 0x00000000, 0x00000000
 
 /**
- * ARGS
- *   s32 entries (e)
- * FORMAT
- *   0000007C eeeeeeee
- *   size = 0x8
+ * Declares a list of `CS_FADE_OUT_SEQ` entries.
  */
-#define CS_FADE_BGM_LIST(entries) CS_CMD_FADEBGM, CMD_W(entries)
+#define CS_FADE_OUT_SEQ_LIST(entries) \
+    CS_CMD_FADE_OUT_SEQ, CMD_W(entries)
 
 /**
- * ARGS
- *   s16 fadeType (t), s16 startFrame (s), s16 endFrame (e)
- * FORMAT
- *   Capital U is Unused
- *   ttttssss eeeeUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU 00000000 00000000 00000000
- *   size = 0x30
+ * Fades a sequence out over the specified duration.
  */
-#define CS_FADE_BGM(fadeType, startFrame, endFrame, unused0, unused1, unused2, unused3, unused4, unused5, unused6, unused7) \
-    CMD_HH(fadeType, startFrame), CMD_HH(endFrame, unused0), \
+#define CS_FADE_OUT_SEQ(seqPlayer, startFrame, endFrame, unused0, unused1, unused2, unused3, unused4, unused5, unused6, unused7) \
+    CMD_HH(seqPlayer, startFrame), CMD_HH(endFrame, unused0), \
     CMD_W(unused1), CMD_W(unused2), CMD_W(unused3), CMD_W(unused4), CMD_W(unused5), \
     CMD_W(unused6), CMD_W(unused7), 0x00000000, 0x00000000, 0x00000000
 
 /**
- * ARGS
- *   s32 entries (e)
- * FORMAT
- *   0000008C eeeeeeee
- *   size = 0x8
+ * Declares a list of `CS_TIME` entries.
  */
-#define CS_TIME_LIST(entries) CS_CMD_SETTIME, CMD_W(entries)
+#define CS_TIME_LIST(entries) \
+    CS_CMD_SETTIME, CMD_W(entries)
 
 /**
- * ARGS
- *   s16 unk (u), s16 startFrame (s), s16 endFrame (e), s8 hour (h), s8 min (m)
- * FORMAT
- *   Capital U is Unused
- *   uuuussss eeeehhmm UUUUUUUU
- *   size = 0xC
+ * Sets the time of day.
+ * Both the day time and skybox time are set by this command.
  */
-#define CS_TIME(unk, startFrame, endFrame, hour, min, unused) \
-    CMD_HH(unk, startFrame), \
-    CMD_HBB(endFrame, hour, min), \
-    CMD_W(unused)
+#define CS_TIME(unk, frame, unused0, hour, min, unused1) \
+    CMD_HH(unk, frame), \
```

Comment:
btw why do some of these have "unk" instead of "unused"
(where the structs typically have "unused")